pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
+line-length = 100
+exclude = '''
+/(
+  | \.git
```

Comment:
From a pants perspective, pants already ignores these files when it expands globs; are they here in order to preserve the ability to just run `black` not via pants? If so, may be worth a comment to that effect